### ISSUE TITLE
Missing AppArmor rule on fontawesome-webfont.woff2

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#2024]
## Description

There is a missing AppArmor rule on one of the web fonts we are using on the journalist interface. 

## Steps to Reproduce

1. Install SecureDrop 0.4
2. Visit journalist interface
3. Watch `/var/log/apache2/journalist-error.log` (and/or watch your emails for the OSSEC alarms)

## Expected Behavior

No unusual alerts occur or lines appear in the logs. 

## Actual Behavior

Admins will get errors due to a missing AppArmor rule for `/var/www/securedrop/static/fonts/fontawesome-webfont.woff2` in their logs as well as the following OSSEC alert:

```
OSSEC HIDS Notification.
2017 Jul 21 14:07:59

Received From: (app) 10.20.2.2->/var/log/syslog
Rule: 100012 fired (level 7) -> "Apparmor denied event"
Portion of the log(s):

Jul 21 14:07:57 app kernel: [ 9892.514406] audit: type=1400 audit(1500671277.745:21): apparmor="DENIED" operation="open" profile="/usr/sbin/apache2" name="/var/www/securedrop/static/fonts/fontawesome-webfont.woff2" pid=1319 comm="apache2" requested_mask="r" denied_mask="r" fsuid=33 ouid=33

 --END OF NOTIFICATION

OSSEC HIDS Notification.
2017 Jul 21 14:07:59

Received From: (app) 10.20.2.2->/var/log/apache2/journalist-error.log
Rule: 1002 fired (level 2) -> "Unknown problem somewhere in the system."
Portion of the log(s):

[Fri Jul 21 14:07:57.748498 2017] [core:error] [pid 1317:tid 2984486479616] (13)Permission denied: [client 127.0.0.1:53268] AH00132: file permissions deny server access: /var/www/securedrop/static/fonts/fontawesome-webfont.woff2, referer: http://[redacted].onion/static/css/journalist.css
```

## Comments

It looks like this font was left out of the AppArmor rules.

**Comment by dachary** [comment]
I ran into this as well.

**Comment by conorsch** [comment]
The error is also shown in the web console for the Journalist Interface. We don't have any testing to catch such errors. Adding them via Selenium or similar would be difficult, since I'm not aware of any browser APIs to expose the events in the web console.

Nearer term, we could add config tests after the integration tests to check for log lines such as:

```
[Sat Jul 22 02:10:21.430104 2017] [core:error] [pid 3582:tid 3491550607104] (13)Permission denied: [client 127.0.0.1:56813] AH00132: file permissions deny server access: /var/www/securedrop/static/fonts/fontawesome-webfont.woff2, referer: http://6dtblbz3uwxusi2w.onion/static/css/journalist.css
```

**Comment by redshiftzero** [comment]
Yep automated testing for this would be excellent. Fortunately(?) our extremely sensitive OSSEC alerts will start screaming from the rooftops when unusual error messages appear in logs, but we shouldn't be relying on that. 

Fixed by #2025, closing.
